server/configuration.go

```diff
@@ -90,7 +94,19 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(err, "failed to load plugin configuration")
 	}
 
-	p.setConfiguration(configuration)
+	r := regexp.MustCompile("^https?://(.*?)/")
```

Comment:
Can we move this out to a function like `stripProtocol`? 
Also a simpler way to do this might be to use https://golang.org/pkg/net/url/#Parse then just call https://golang.org/src/net/url/url.go?s=29480:29511#L1051 might also cover cases that haven't been covered here.

Reply:
That's a good idea. But then I read that `Trying to parse a hostname and path without a scheme is invalid but may not necessarily return an error, due to parsing ambiguities.` So I'm not sure if it will work for us here. What do you think?

I'll move it into `parseHostFromUrl` because it also strips the path.

I'm 2/5 that we need it at all, since the instructions say to only give a hostname. But if we don't parse it, the links we create will fail silently, and that's worse in my mind. We'd have to parse or verify somehow anyway.

Reply:
Huh. Didn't know that. Good thing you read the docs! I think it's fine how it is then.